packages/cli/src/lib/new/factories/backendModule.ts

```diff
@@ -100,5 +100,9 @@ export const backendModule = createFactory<Options>({
       cwd: targetDir,
       optional: true,
     });
+    await Task.forCommand('yarn backstage-cli repo fix', {
```

Comment:
Hmm, I think this we can assume that this is safe to run, can be pretty disruptive if the repo isn't already in a good state.

Perhaps this is behind a `--fix` flag for `new`?

Reply:
Hm how do you mean - that it's not safe? It only runs for the new thing. I thought that this would silently move on as well if it failed.

Reply:
It might apply a lot of changes to the repo that are unrelated to the new package

Reply:
Oh in all honesty I thought that it wouldn't do the full repo when cwd:d into the plugin dir. Maybe I can do something about that by being a bit smarter, I'll look into it

Reply:
The `repo` commands always run across the entire repo, so it'd need something like a `package fix` command